src/com/cft/contactmerge/contact/tests/NameTests.java

```diff
@@ -180,4 +180,31 @@ void Name_setParseSuffixAndPrefixInLastName_PunctuatedLastName(){
         assertEquals("Mr.", personOne.getPrefix(), testFailedMessage("Mr.", personOne.getPrefix()));
         assertEquals("Jr.", personOne.getSuffix(), testFailedMessage("Jr.", personOne.getSuffix()));
     }
+
+    @Test
+    void Name_isMiddleNameInFullName_True_PunctuationIgnoreSpaces(){
+        Name name = new Name();
+
+        assertTrue(name.isMiddleNameInFullName("Mr.Page,Christopher,James,1st"));
+        assertTrue(name.isMiddleNameInFullName("Mr. Page, Christopher, James, I"));
```

Comment:
Using GiftWorks as a test, I'm finding that it won't generate/suggest use of the prefix or suffix when you use the last name first format. It suggests: "Mr. Page", "Page, James", "Page, James Christopher", or "Page, James C.". As I've mentioned before, I wouldn't worry about handling unknown cases. Let's wait until we encounter something and then deal with it when we have a real example.

Reply:
Okay, I will remove these test cases